dipy/reconst/dti.py

```diff
@@ -82,6 +82,61 @@ def fractional_anisotropy(evals, axis=-1):
 
     return fa
 
+def geodesic_anisotropy(evals, axis=-1):
+    r"""
+    Geodesic anisotropy (FA) of a diffusion tensor.
+
+    Parameters
+    ----------
+    evals : array-like
+        Eigenvalues of a diffusion tensor.
+    axis : int
+        Axis of `evals` which contains 3 eigenvalues.
+
+    Returns
+    -------
+    ga : array
+        Calculated GA. In the range [0, +infty)
+
+    Notes
+    --------
+    GA is calculated using the following equation [1]_ and [2]_:
+
+    .. math::
+
+        GA = \sqrt{\sum_{i=1}^3 \log^2{\left ( \lambda_i/<\mathbf{D}> \right )}, 
```

Comment:
I think there's a curly brace missing here. I believe this should be: 

A = \sqrt{\sum_{i=1}^3 \log^2{\left ( \lambda_i/<\mathbf{D}> \right )}},\quad \textrm{where} \quad <\mathbf{D}> = \frac{\lambda_1+\lambda_2+\lambda_3}{3}